android/src/main/java/com/google/samples/apps/iosched/droidconitaly15/JsonTransformer.java

```diff
@@ -388,8 +393,7 @@ private static String convertDateTime(
      * From html to plain text, yo.
      */
     @Nullable
-    private static String escapeHtml(@Nullable String html)
-    {
+    private static String escapeHtml(@Nullable String html) {
         return html == null ? null : Html.fromHtml(html).toString(); // THE HORROR
```

Comment:
It's made like that on purpose: if the original text is null, I don't want to "add" information